README.md

```diff
@@ -37,13 +37,13 @@ We also have moved some of the library to their own toplevel package, like:
 - Version helper: [fortio.org/version](https://github.com/fortio/version#version)
 - CLI helpers integrating the above to reduce toil making new tools [fortio.org/cli](https://github.com/fortio/cli#cli) and servers [fortio.org/scli](https://github.com/fortio/scli#scli) for arguments, flags, usage, dynamic config, etc...
 
-If you want to connect to fortio using https and fortio to provide real TLS certificates, or to multiplex grpc and regular http behind a single port, check out [Fortio Proxy](https://github.com/fortio/proxy#fortio-proxy).
+If you want to connect to fortio using HTTPS and fortio to provide real TLS certificates, or to multiplex gRPC and regular HTTP behind a single port, check out [Fortio Proxy](https://github.com/fortio/proxy#fortio-proxy).
```

Comment:
thanks!

https and http I'm not sure... it "screams" a bit in English and has become somewhat of a noun ?
(gRPC ok)

Reply:
They are acronym

Hypertext Transfer Protocol Secure (HTTP): https://en.wikipedia.org/wiki/HTTP
Hypertext Transfer Protocol Secure (HTTPS): https://en.wikipedia.org/wiki/HTTPS

They might be screaming, but they are expected to be acronym.

I can discard them if you want no problem anyway.

please finish the review and tell me